index.html

```diff
@@ -0,0 +1,361 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Twitter</title>
+
+</head>
+
+<body>
+    <main>
+        <div class="left-column">
+            <div class="left-navigation">
+                <div class="left-twitter nav-button">
+                    <a href="#" class="nav-hover">
+                        <img src="assets\twitter-color-icon.svg" alt="Twitter Icon">
```

Comment:
The way you structure your divs are very presentable and easy to use. Makes it easy to know where youre focusing on